docs/source/quickstart.mdx

```diff
@@ -195,6 +195,23 @@ That's it! Running the Router with the `--dev` flag enables a development mode t
 
 Visit `http://127.0.0.1:4000` to open Apollo Sandbox, inspect your entire supergraph, and run your first queries!
 
+**Do not use development mode in production!**
+
+Add the `--dev` flag is the equivalent runnning `./router --hot-reload` with this configuration:
+
+```yaml
+sandbox:
+  enabled: true
+supergraph:
+  introspection: true
+include_subgraph_errors:
+  all: true
+plugins:
+  experimental.expose_query_plan: true
+```
+
+If you need some of these features enabled in production, specify them directly in your configuration instead of using `--dev`.
```

Comment:
````suggestion
<blockquote>


**⚠️ Do not use the `--dev` flag in production!** If you want to replicate any development mode functionality in production, modify your YAML configuration file instead.

Using `--dev` is equivalent to running `./router --hot-reload` with the following configuration:

```yaml
sandbox:
  enabled: true
supergraph:
  introspection: true
include_subgraph_errors:
  all: true
plugins:
  experimental.expose_query_plan: true
```


</blockquote>
````

Reply:
(The extra spaces after the `blockquote` I believe are necessary because MDX is unhappy if markdown formatting appears too close to a block-level HTML element)